controllers/comment.js

```diff
@@ -0,0 +1,23 @@
+import Sequelize from 'sequelize';
+import models from '../models';
+
+const { iLike } = Sequelize.Op;
+const { Article, Comment } = models;
+
+export default {
+  async addComment(req, res) {
+    const article = await Article.findOne({
+      where: { slug: { [iLike]: req.params.slug } }
+    });
+    const comment = await Comment.create({
```

Comment:
You probably want to name this `comment` something else. `comment.comment` isn't good for aesthetics.